Subo CLI Segfaults and Panics During Bundle

It is an amazing work you folks are doing here. I encountered an issue while following the walkthrough [here](https://www.youtube.com/watch?v=d0UT5WghFpQ). 

When I got to `subo build helloworld --docker`,  the following error shows up:

```
...
✅ DONE: helloworld was built -> /Users/.../Desktop/projects/important-api/helloworld/helloworld.wasm
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x13b1670]

goroutine 1 [running]:
github.com/suborbital/subo/subo/command.BuildCmd.func1(0xc000130780, 0xc0000ec4e0, 0x1, 0x2, 0x0, 0x0)
	/private/tmp/subo-20210905-4022-iy0t5p/subo-0.0.22/subo/command/build.go:81 +0x310
github.com/spf13/cobra.(*Command).execute(0xc000130780, 0xc0000ec4c0, 0x2, 0x2, 0xc000130780, 0xc0000ec4c0)
	/private/tmp/subo-20210905-4022-iy0t5p/subo-0.0.22/vendor/github.com/spf13/cobra/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000c0f00, 0x0, 0xffffffff, 0xc000096058)
	/private/tmp/subo-20210905-4022-iy0t5p/subo-0.0.22/vendor/github.com/spf13/cobra/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/private/tmp/subo-20210905-4022-iy0t5p/subo-0.0.22/vendor/github.com/spf13/cobra/command.go:897
main.main()
	/private/tmp/subo-20210905-4022-iy0t5p/subo-0.0.22/subo/main.go:10 +0x2a
```

No `runnables.wasm.zip` generated.

Subo CLI v0.0.22 (Installed via brew)
macOS Catalina v10.15.6

--- a/subo/command/build.go
+++ b/subo/command/build.go
@@ -28,7 +28,7 @@ func BuildCmd() *cobra.Command {
 			}
 
 			if len(bdr.Context.Runnables) == 0 {
-				return errors.New("🚫 no runnables found in current directory (no .runnable yaml files found)")
+				return errors.New("🚫 no runnables found in current directory (no .runnable.yaml files found)")
 			}
 
 			if bdr.Context.CwdIsRunnable {
@@ -41,6 +41,10 @@ func BuildCmd() *cobra.Command {
 			noBundle, _ := cmd.Flags().GetBool("no-bundle")
 			shouldBundle := !noBundle && !bdr.Context.CwdIsRunnable && len(langs) == 0
 			shouldDockerBuild, _ := cmd.Flags().GetBool("docker")
+
+			if bdr.Context.CwdIsRunnable && shouldDockerBuild {
+				return errors.New("🚫 cannot build Docker image for a single Runnable (must be a project)")
+			}
 
 			useNative, _ := cmd.Flags().GetBool("native")
 			makeTarget, _ := cmd.Flags().GetString("make")
@@ -94,7 +98,11 @@ func BuildCmd() *cobra.Command {
 				defer util.LogDone(fmt.Sprintf("bundle was created -> %s @ %s", bdr.Context.Bundle.Fullpath, bdr.Context.Directive.AppVersion))
 			}
 
-			if shouldDockerBuild {
+			if shouldDockerBuild && !bdr.Context.CwdIsRunnable {
+				if bdr.Context.Directive == nil {
+					return errors.New("🚫 failed to open Directive.yaml")
+				}
+
 				os.Setenv("DOCKER_BUILDKIT", "0")
 
 				if _, err := util.Run(fmt.Sprintf("docker build . -t=%s:%s", bdr.Context.Directive.Identifier, bdr.Context.Directive.AppVersion)); err != nil {

